### DISCUSSION TITLE
Chia Helper (Renderer)

### DISCUSSION THREAD

**Discussion by chris92q** [Chia-Network/chia-blockchain#4710] [Support]
Hi there,

What does the above mentioned process do?
It uses much processing power on my little MacBook Air and causes drives to respond a little slow! Above 5sec...

**Comment by ilyakooo0** [comment]
I think this an [electron](https://www.electronjs.org) process that actually renders the GUI.

**Reply by forrestcamps** [reply]
What can I do? I'm facing this problem as well   : (

**Comment by chris92q** [comment]
Puh... But why does this ramp up the CPU to 50ish%? 
The GUI isn't really komplex...

**Reply by forrestcamps** [reply]
@chris92q I'm facing the same problem, after I did the upgrade to 1.1.5 version, I'm nervous about this, is slower and I'm  afraid with the temperature 90º Celsiu . My macbook pro is i7 16gb ram  2017. This Chia helper (renderer) is using my CPU extremely 60%. Do you already solve the problem? Can you help me? (Marcos from Brazil)  Anyone can help me please?

**Reply by ilyakooo0** [reply]
I can assure you that your MacBook will manage the thermals just fine. Nothing bad will happen even if you rub it at 100% CPU for hours. The CPU will start running slower and the fan will run faster to compensate. 

The manufacturer will not allow you to physically damage the MacBook by running a CPU-intensive program.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
